mkstage4.sh

```diff
@@ -80,7 +93,7 @@ shift;OPTIONS="$@"
 EXCLUDES="\
 --exclude=.bash_history \
 --exclude=dev/* \
---exclude=lost+found \
+--exclude=lost+found/* \
```

Comment:
so why would we need to archive the empty directory? It's possible the image will be deployed on non-ext4 later, in which case the lost+found directory will be aberrant. conversely, if the system is deployed on an ext4 system, there will already be a lost+found folder...